### DISCUSSION TITLE
Bigger Application tutorial: missing installation / running instructions

### DISCUSSION THREAD

**Discussion by carschno** [fastapi/fastapi#6775] [Questions] [answered]
### First check

* [x] I added a very descriptive title to this issue.
* [x] I used the GitHub search to find a similar issue and didn't find it.
* [x] I searched the FastAPI documentation, with the integrated search.
* [x] I already searched in Google "How to X in FastAPI" and didn't find any information.
* [x] I already read and followed all the tutorial in the docs and didn't find an answer.
* [x] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
* [x] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
* [x] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).
* [x] After submitting this, I commit to one of:
    * Read open issues with questions until I find 2 issues where I can help someone and add a comment to help there.
    * I already hit the "watch" button in this repository to receive notifications and I commit to help at least 2 people that ask questions in the future.
    * Implement a Pull Request for a confirmed bug.

### Description

The [Bigger Applications tutorial](https://fastapi.tiangolo.com/tutorial/bigger-applications/) explains how to set up a FastApi project with multiple files and sub-packages. However, it remains unclear how this is supposed to be run.

I try to run the example like this:

    fastapi/docs_src/bigger_applications/app$ uvicorn main:app --reload

This results in the following error:

```
[...]
  File "./main.py", line 3, in <module>
    from .dependencies import get_query_token, get_token_header
ImportError: attempted relative import with no known parent package
```

This is to be expected as the (sub-)packages are not installed. However, it remains unclear how this is expected to be installed as a package, and how it is to be run.

### Environment

* OS: Linux
* FastAPI Version: 0.63.0

* Python version: 3.9.1

**Comment by insomnes** [comment, marked as answer]
@carschno hello!
You should run it (cause of [relative imports](https://fastapi.tiangolo.com/tutorial/bigger-applications/#how-relative-imports-work)) as mentioned in [docs](https://fastapi.tiangolo.com/tutorial/bigger-applications/#check-the-automatic-api-docs), from `docs_src/bigger_applications`
as 
`uvicorn app.main:app --reload`

**Comment by tiangolo** [comment]
Thanks for the help here @insomnes ! 👏  🙇 

Thanks for reporting back and closing the issue @carschno  👍

> Sorry for the long delay! 🙈  I wanted to personally address each issue/PR and they piled up through time, but now I'm checking each one in order.
